src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -96,13 +96,15 @@ void MediaManagerImpl::set_mock_mic_recorder(MediaAdapterImpl* media_adapter) {
 
 #endif  // EXTENDED_MEDIA_MODE
```

Comment:
@indlu I am just curious why this macro is absent from the header file, IMO it will break the build if it is turned off. Probably all related tests might have the same issue.